samples/advanced-markers-graphics/index.ts

```diff
@@ -1,96 +1,109 @@
-    // A marker with a with a URL pointing to a PNG.
-    const beachFlagImg = document.createElement('img');
-    beachFlagImg.src = new URL('./public/beachflag.png', import.meta.url).href;
+  const pinSvg = parser.parseFromString(
+    pinSvgString,
+    'image/svg+xml'
+  ).documentElement;
+  const pinSvgMarkerView = new AdvancedMarkerElement({
+    position: { lat: 37.42475, lng: -122.094 },
+    content: pinSvg,
+    title: 'A marker using a custom SVG image.',
+    //@ts-ignore
+    anchorLeft: '-50%',
+    anchorTop: '-50%',
+  });
+  mapElement.append(pinSvgMarkerView);
+  // [END maps_advanced_markers_graphics_inline]
 
-    const beachFlagMarkerView = new AdvancedMarkerElement({
-        map,
-        position: { lat: 37.434, lng: -122.082 },
-        content: beachFlagImg,
-        title: 'A marker using a custom PNG Image',
-        //@ts-ignore
-        anchorLeft: "0px",
-        anchorTop: "100%",
-    });
-    // [END maps_advanced_markers_graphics_png]
+  // [START maps_advanced_markers_graphics_png]
+  // A marker with a with a URL pointing to a PNG.
+  const beachFlagImg = document.createElement('img');
+  beachFlagImg.src = new URL('./public/beachflag.png', import.meta.url).href;
 
-    // [START maps_advanced_markers_graphics_svg_glyph]
-    // A marker with a custom SVG glyph.
-    const glyphImgSrc =  new URL('./public/google_logo_g.svg', import.meta.url);
-    
-    const glyphSvgPinElement = new PinElement({
-        //@ts-ignore
-        glyphSrc: glyphImgSrc,
-    });
+  const beachFlagMarkerView = new AdvancedMarkerElement({
+    position: { lat: 37.434, lng: -122.082 },
+    content: beachFlagImg,
+    title: 'A marker using a custom PNG Image',
+    //@ts-ignore
+    anchorLeft: '0px',
+    anchorTop: '100%',
+  });
+  mapElement.append(beachFlagMarkerView);
+  // [END maps_advanced_markers_graphics_png]
 
-    const glyphSvgMarkerView = new AdvancedMarkerElement({
-        map,
-        position: { lat: 37.425, lng: -122.07 },
-        content: glyphSvgPinElement.element,
-        title: "A marker using a custom SVG for the glyph.",
-    });
-    // [END maps_advanced_markers_graphics_svg_glyph]
+  // [START maps_advanced_markers_graphics_svg_glyph]
+  // A marker with a custom SVG glyph.
+  const glyphImgSrc = new URL('./public/google_logo_g.svg', import.meta.url);
 
-    // [START maps_advanced_markers_graphics_place_icon]
-    // A marker customized using a place icon and color, name, and geometry.
-    const place = new Place({
-        id: 'ChIJN5Nz71W3j4ARhx5bwpTQEGg',
-    });
+  const glyphSvgPinElement = new PinElement({
+    //@ts-ignore
+    glyphSrc: glyphImgSrc,
+  });
 
-    // Call fetchFields, passing the desired data fields.
-    await place.fetchFields({ fields: ['location', 'displayName', 'svgIconMaskURI', 'iconBackgroundColor'] });
+  const glyphSvgMarkerView = new AdvancedMarkerElement({
+    position: { lat: 37.425, lng: -122.07 },
+    content: glyphSvgPinElement.element,
```

Comment:
pin is a web component, so we can simply do:
glyphSvgMarker.append(glyphSvgPinElement);